code/modules/admin/verbs/adminhelp.dm

```diff
@@ -567,42 +569,45 @@ GLOBAL_DATUM_INIT(ahelp_tickets, /datum/admin_help_tickets, new)
-	dat += "[FOURSPACES][TicketHref("Refresh", ref_src)][FOURSPACES][TicketHref("Re-Title", ref_src, "retitle")]"
-	if(state != AHELP_ACTIVE)
-		dat += "[FOURSPACES][TicketHref("Reopen", ref_src, "reopen")]"
-	dat += "<br><br>Opened at: [gameTimestamp(wtime = opened_at)] (Approx [DisplayTimeText(world.time - opened_at)] ago)"
-	if(closed_at)
-		dat += "<br>Closed at: [gameTimestamp(wtime = closed_at)] (Approx [DisplayTimeText(world.time - closed_at)] ago)"
-	dat += "<br><br>"
-	if(initiator)
-		dat += "<b>Actions:</b> [FullMonty(ref_src)]<br>"
-	else
-		dat += "<b>DISCONNECTED</b>[FOURSPACES][ClosureLinks(ref_src)]<br>"
-	dat += "<br><b>Log:</b><br><br>"
-	for(var/I in ticket_interactions)
-		dat += "[I]<br>"
-
-	// Append any tickets also opened by this user if relevant
-	var/list/related_tickets = GLOB.ahelp_tickets.TicketsByCKey(initiator_ckey)
-	if (related_tickets.len > 1)
-		dat += "<br/><b>Other Tickets by User</b><br/>"
-		for (var/datum/admin_help/related_ticket in related_tickets)
-			if (related_ticket.id == id)
-				continue
-			dat += "[related_ticket.TicketHref("#[related_ticket.id]")] ([related_ticket.ticket_status()]): [related_ticket.name]<br/>"
-	dat += "</html>"
-
-	usr << browse(dat.Join(), "window=ahelp[id];size=750x480")
+	// MONKESTATION EDIT START
+	ui_interact(usr)
+	// var/list/dat = list("<html><head><meta http-equiv='Content-Type' content='text/html; charset=UTF-8'><title>Ticket #[id]</title></head>")
+	// var/ref_src = "[REF(src)]"
+	// dat += "<h4>Admin Help Ticket #[id]: [LinkedReplyName(ref_src)]</h4>"
+	// dat += "<b>State: [ticket_status()]</b>"
+	// dat += "[FOURSPACES][TicketHref("Refresh", ref_src)][FOURSPACES][TicketHref("Re-Title", ref_src, "retitle")]"
+	// if(state != AHELP_ACTIVE)
+	// 	dat += "[FOURSPACES][TicketHref("Reopen", ref_src, "reopen")]"
+	// dat += "<br><br>Opened at: [gameTimestamp(wtime = opened_at)] (Approx [DisplayTimeText(world.time - opened_at)] ago)"
+	// if(closed_at)
+	// 	dat += "<br>Closed at: [gameTimestamp(wtime = closed_at)] (Approx [DisplayTimeText(world.time - closed_at)] ago)"
+	// dat += "<br><br>"
+	// if(initiator)
+	// 	dat += "<b>Actions:</b> [FullMonty(ref_src)]<br>"
+	// else
+	// 	dat += "<b>DISCONNECTED</b>[FOURSPACES][ClosureLinks(ref_src)]<br>"
+	// dat += "<br><b>Log:</b><br><br>"
+	// for(var/I in ticket_interactions)
+	// 	dat += "[I]<br>"
+
+	// // Append any tickets also opened by this user if relevant
+	// var/list/related_tickets = GLOB.ahelp_tickets.TicketsByCKey(initiator_ckey)
+	// if (related_tickets.len > 1)
+	// 	dat += "<br/><b>Other Tickets by User</b><br/>"
+	// 	for (var/datum/admin_help/related_ticket in related_tickets)
+	// 		if (related_ticket.id == id)
+	// 			continue
+	// 		dat += "[related_ticket.TicketHref("#[related_ticket.id]")] ([related_ticket.ticket_status()]): [related_ticket.name]<br/>"
+	// dat += "</html>"
+
+	// usr << browse(dat.Join(), "window=ahelp[id];size=750x480")
+	// MONKESTATION EDIT END
```

Comment:
A little tip: DM supports `/* */` for block comments. It's much easier to just use that, than to prepend `//` onto every line.

Reply:
Its not that much easier than telling vs code to block comment, but I can see how it would make porting stuff easier, so I'll convert it to a block